tools/system_libs.py

```diff
@@ -2047,6 +2047,7 @@ def can_use(self):
 class libstubs(DebugLibrary):
   name = 'libstubs'
   src_dir = 'system/lib/libc'
+  includes = ['system/lib/libc/musl/src/include']
```

Comment:
This PR doesn't add new `#include`s - is this addition necessary?

Reply:
Its so that system/lib/libc/emscripten_syscall_stubs.c doesn't need to redefine `weak` that musl's features.h already provides.

Reply:
But how does that `features.h` get included, if there isn't a new `#include <features.h>` in this PR?

Reply:
features.h is gets included transitively, and there is public and a private copy.  In this case just adding the internal include path gives access to musl-internal macros like this.

Reply:
I see, wow, that is surprising but I guess it make sense.